sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ActiveClientTokenManagerTest.java

```diff
@@ -22,18 +24,29 @@
 import java.time.Duration;
 import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
+import java.util.concurrent.atomic.AtomicInteger;
 
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 
 class ActiveClientTokenManagerTest {
     private static final String AUDIENCE = "an-audience-test";
     private static final String SCOPES = "scopes-test";
-    private static final Duration TIMEOUT = Duration.ofSeconds(4);
+    private static final Duration DEFAULT_DURATION = Duration.ofSeconds(20);
 
     @Mock
     private ClaimsBasedSecurityNode cbsNode;
 
+    @BeforeAll
+    static void beforeAll() {
+        StepVerifier.setDefaultTimeout(Duration.ofSeconds(30));
```

Comment:
Does setting the default timeout change the timeout for all tests running outside this class too?

Reply:
No. I have an `AfterAll` notation to reset the timeout.